src/components/dashboard/FaceVerification/screens/VerificationScreen.js

```diff
@@ -49,11 +53,11 @@ const FaceVerification = ({ screenProps }) => {
     hideLoading()
 
     // firing event
-    fireEvent(FV_GETREADY_ZOOM)
+    fireEvent(`${FV_GETREADY_ZOOM}_AB_${AB}`)
```

Comment:
we AB test only FV_INTRO and FV_INSTRUCTIONS
and put the AB as parameters

Reply:
done